pkg/controller/worker/machines.go

```diff
@@ -55,6 +56,12 @@ func (w *workerDelegate) DeployMachineClasses(ctx context.Context) error {
 			return err
 		}
 	}
+
+	// Delete any older version machine class CRs.
```

Comment:
I guess this code is just temporarily required? Can we add a TODO to remind us to remove it in a few releases?